### PR TITLE
copyKeys

### PR DESCRIPTION
**Pull request by FelipeParreira**
[#516](https://github.com/char0n/ramda-adjunct/issues/516)

### CODE REVIEW

**Comment by char0n** [comment]
Closing this due to unresponsiveness of author in this PRs.

**Comment by FelipeParreira** [comment]
Hey, @char0n sorry for my unresponsiveness. I can take a look at this PR again. What are the issues?

**Comment by char0n** [comment]
Reopening and doing CR, glad to hear from you ;]

File: src/copyKeys.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { curryN } from 'ramda';
+
+import renameKeys from './renameKeys';
+
+/**
+ * Creates a new object with the own properties of the provided object, and the
+ * keys copied according to the keysMap object as `{oldKey: newKey}`.
+ * When some key is not found in the keysMap, then it's passed as-is.
```
Comment (char0n) [inline src/copyKeys.js:8]:
```suggestion
 * When key is not found in the keysMap, then a shallow clone of an object is returned.
```

File: types/index.d.ts [hunk 2]
```diff
@@ -594,6 +594,14 @@ declare namespace RamdaAdjunct {
         renameKeys(keysMap: Dictionary<string>, obj: object): object;
         renameKeys(keysMap: Dictionary<string>): (obj: object) => object;
 
+        /**
+         * Creates a new object with the own properties of the provided object, and the
+         * keys copied according to the keysMap object as `{oldKey: newKey}`.
+         * When some key is not found in the keysMap, then it's passed as-is.
```
Comment (char0n) [inline types/index.d.ts:600]:
```suggestion
         * When no key from the keysMap is found, then a shallow clone of an object is returned.
```

**Review by char0n — APPROVED** [review]
Added couple of changes and additional tests. This is good to go! Thanks

**Pull request merged**